### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by lerfast**
Hello fellow code reviewer, I did the following tasks:

 I create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:

List all books.
List all people.
Create a person (teacher or student, not a plain Person).
Create a book.
Create a rental.
List all rentals for a given person id.

I create main.rb that define the entry point, this will be a method called main that is invoked at the end of the file. This method do the following:

Present the user with a list of options to perform.
Lets users choose an option.
If needed, ask for parameters for the option.
Have a way to quit the app.

Here's the screenshots:

image

image

image

image

image

image

image

### CODE REVIEW

File: app.rb
Comment (Alusp) [inline app.rb:1]: - [ ] Good job so far, :clap: however, on your, there is a little mismatch on the list of people, te name is showing integer value while the age is showing the names, please resolve it. :slightly_smiling_face: 
 Below is the screenshot showing the mismatch, please pay attention on the red markings: :arrow_down: 
Namin

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @lerfast, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

 ## Highlights
✔️ Build interactive console apps. ✔️
✔️ Lets users choose an option. ✔️
✔️ Tasks almost complete  ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb
Comment (lerfast) [reply app.rb:1]: Thank you for notice that, I fixed it!!!

image

**Review by levy002 — APPROVED** [review]
## Hello @lerfast  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter error :heavy_check_mark: 
- You implemented the suggested changes :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**